src/chanjo2/crud/samples.py

```diff
@@ -74,3 +75,11 @@ def create_sample_in_case(db: Session, sample: SampleCreate) -> Optional[SQLSamp
     db.commit()
     db.refresh(db_sample)
     return db_sample
+
+
+def remove_sample(db: Session, sample_name: str) -> int:
```

Comment:
```suggestion
def delete_sample(db: Session, sample_name: str) -> int:
```
I hink it is good to keep to the `CRUD` verbs if possible